sdk/rust-examples/prime-numbers-generator/generate-set/src/main.rs

```diff
@@ -0,0 +1,39 @@
+//! Prime Numbers Generator: Generate the number set
+//!
+//! ## Context
+//!
+//! Generate all numbers from 2 to 120 and write them to a text file
```

Comment:
Can we make this upper bound configurable, taken from a command line parameter?  So e.g.,

```
./generate-set --upper-bound 256
```

generates all numbers from 1 to 256?  You can use the `clap` library for command line parsing (the `freestanding-execution-engine` uses it, so you can see examples of how it is done there).

Reply:
I will try it

Reply:
I used a direct environment variables to be passed to the FEE. I thought this would be easier to pass to FEE. If `clap` is preferred I will try to figure it out.